pocs/utils/db/postgres.py

```diff
@@ -141,25 +148,38 @@ def add_header_to_db(header, conn=None, logger=None):
         'exp_time': header['EXPTIME'],
         'ra_rate': header['RA-RATE'],
         'pocs_version': header['CREATOR'],
-        'piaa_state': header['PSTATE'],
+        'piaa_state': header.get('PSTATE', 'initial'),
```

Comment:
Why are we using get sometimes, and sometimes not?

Reply:
When I want to set a default if not provided. The other cases should probable be handled better.